### DISCUSSION TITLE
Add multi-language support

### DISCUSSION THREAD

**Discussion by hicupalot** [Chatterino/chatterino2#3821] [Ideas]
Hello! I think adding multi-language support would benefit Chatterino greatly! You could do this through Crowdlin or some other language support means, doing it through Crowdlin would allow you to quickly add new language as it would be crowdsourced.

**Comment by Mm2PL** [comment]
This is a huge undertaking to make the app translatable and to translate it. Even with crowd sourcing. I don't think any of the developers of this app care for this. I could be wrong though. I certainly prefer having it in English.

Currently we don't even support rendering any RTL languages in chat.

Maybe in the future. We still have a long way to go.

15 cze 2022 08:39:26 hicupalot ***@***.***>:

**Reply by pajlada** [reply]
As Mm2PL said, big undertaking and none of the core developers currently have this need so no internal push for this functionality is being done.

We would happily accept PRs that help make Chatterino multi-language friendly. I think Qt has a bunch of things for this already so no new external library would be needed.
